File: flax_search_service/docs/tutorial/php/t4_show_db_info.php
<?php
include_once "flax/flaxclient.php";

# open a connection to Flax Server
$conn = new FlaxSearchService('http://localhost:8080');

# get a reference to the database
$db = $conn->getDatabase('books');

# print the info
print $db->getDocCount() ." documents in 'books'\n";

?>

File: flax_search_service/docs/tutorial/perl/t4_show_db_info.pl
#!/usr/bin/perl

use strict;
use warnings;
use Error qw(:try);

use Flax;
use Flax::Error;
use Flax::Field;
use Flax::SummarySettings;

my $flax = Flax->new('url'=>'http://localhost:8080');
my $db = $flax->getDatabase('books');

# print the info
print $db->getDocCount ." documents in 'books'\n";
